### PR TITLE
2nd change

### PR DESCRIPTION
**Pull request by rrperkit**
Aaddsafdfdas

### CODE REVIEW

File: githubteacher-rrperkit.md [hunk 1]
```diff
@@ -1,34 +1,9 @@
-# GitHub for Developers
+# DC USA
```
Comment (rrperkit) [inline githubteacher-rrperkit.md:1]: Testing comments

Comment (rrperkit) [inline githubteacher-rrperkit.md:1]: testing comments

**Review by rrperkit — COMMENTED** [review]
ddd

**Review by GPrathyusha — APPROVED** [review]
good job

**Pull request merged**